gdp/small_batch/gdp_small_batch.py

```diff
@@ -128,7 +304,17 @@ def Y_exists_or_not(m, k, j):
 
     # Objective
     def obj_rule(m):
-        return sum(m.alpha[j]*(pe.exp(m.n[j] + m.beta[j]*m.v[j])) for j in m.j)
+        """
+        Objective: mininimize the investment cost [$].
+        min z = sum(alpha[j]*(n[j] + beta[j]*v[j])) for j = mixer, reactor, centrifuge
+
+        Args:
+            m: pyomo.ConcreteModel
+
+        Returns:
+            Objective function
```

Comment:
The type is missing (pyomo.Objective). Also write the equation

Reply:
Added the Type and the equation.